src/main/resources/application.properties

```diff
@@ -5,7 +5,7 @@ spring.jackson.default-property-inclusion=non_null
 # Example if build.version=1.2.3-internal becomes Version 0.0.18 (1.2.3-internal)
 build.version=
 
-server.port=8080
+server.port=8089
```

Comment:
Reset this port to default 8080

Reply:
You may revert all changes in this file.

Reply:
Ok